patches/extra/ungoogled-chromium/remove-uneeded-ui.patch

```diff
@@ -1,17 +1,27 @@
-# Removes the Google sign-in sections on the settings page
+# Removes in order:
+# 'Tabs from other devices' entry on the History page sidebar
+# link to Google's accessibility site from the settings page
+# link to Google's help site on the About page
+# Safety check section on the settings page
+# option to offer page translations on the settings page
+# cloud printing entry on the settings page
+# option for password leak detection on the settings page
+# Google sign-in and Anonymized Data Collection sections
+# Advanced Protection Program link on the security settings page
+# Safety Check entry on the side menu on the settings page
+# unneeded elements from the profile menu
+
 --- a/chrome/browser/resources/history/side_bar.html
 +++ b/chrome/browser/resources/history/side_bar.html
-@@ -101,11 +101,13 @@
+@@ -101,11 +101,6 @@
          $i18n{historyMenuItem}
          <paper-ripple></paper-ripple>
        </a>
-+<if expr="false">
-       <a href="/syncedTabs" class="page-item" path="syncedTabs"
-           on-click="onItemClick_">
-         $i18n{openTabsMenuItem}
-         <paper-ripple></paper-ripple>
-       </a>
-+</if>
+-      <a href="/syncedTabs" class="page-item" path="syncedTabs"
+-          on-click="onItemClick_">
+-        $i18n{openTabsMenuItem}
+-        <paper-ripple></paper-ripple>
+-      </a>
```

Comment:
The context lines makes both ways the same amount of work to maintain, so I opted to remove the lines rather than block them out.  It seemed like a better way to show what the change is really doing.